### DISCUSSION TITLE
Why not create burn use for resources to craft shovels?

### DISCUSSION THREAD

**Discussion by Myth727** [sunflower-land/sunflower-land#2030] [Ideas]
Why are we digging with an axe? To create higher burn use on resources make it a shovel. This would resolve some issues when it comes to foreman holders like me. It levels the playing field slightly and creates more burning of resources in the game. 

![Screenshot_20230112_203323_Chrome](https://user-images.githubusercontent.com/96928557/212216731-3f5bc3d2-5a45-414c-b245-9ae88213b1b0.jpg)

**Comment by future2back** [comment]
I didn't understand your idea friend, could you explain better? Thank you.

**Reply by Myth727** [reply]
I'll try to break this down better later if possible.  Just there's no point in us digging with axes, we should be using shovels. Otherwise any foreman holders like myself can remove the dead apple and orange trees for free. Cause we don't require axes. To make it fair it should be shovels, this could help burn more resources depending on how it's Implemented.

**Reply by future2back** [reply]
I got the idea. Most of the ideas I see here are for the interests of the community and not for health. I appreciate you as holder of foreman proposing this idea.

The proposal opens the door to the development of new NFTS and makes the distribution a little bit fairer. I don't know what impact this might have on SFL burning because don't know how many trees would be cut down by these 300 owners. I believe it would be a good practice because no NFT or skill affected the fruit trees. Expecting devs take a look on this.
